### DISCUSSION TITLE
How to downgrade firmware? Aqara Bulb ZNLDP12LM 1.34 to 1.32

### DISCUSSION THREAD

**Discussion by Bastiencc** [Koenkk/zigbee2mqtt#15453] [Devices]
Hello !

Since some days I get sometimes Timeout error with my aqara bulb ZNLDP12LM light.aqcn02 . It seems to happen only when I try to turn off the bulb. I'm not sure but seems that the issue came after updating aqara bulb ZNLDP12LM from 1.32 to 1.34. Is it possible to downgrade firmware?

thanks

**Comment by Bastiencc** [comment]
ok I found here , with local firmware : https://www.zigbee2mqtt.io/guide/usage/ota_updates.html#automatic-checking-for-available-updates

**Reply by WarC0zes** [reply]
Hello @Bastiencc,
I have a timeout problem too, but I'm on firmware 1.23 on my bulbs and would like to update them to 1.32 since you say that the 1.34 bug. Where did you download firmware 1.32?

**Comment by Bastiencc** [comment]
You can update to last version, issues finally do not came from this
firmware

**Reply by WarC0zes** [reply]
Ok, I will test. Thanks for the information.

**Comment by Bastiencc** [comment]
If you need to rollback I can send you other firmware version. Let me know.

**Reply by WarC0zes** [reply]
It's good, I flashed my 14 bulbs in 1.34 and no more timeout worries.
On the other hand I can't select the type of power_on_ behavior, I have an error when I select one. Do you have the same problem?
![aqara bulb power](https://user-images.githubusercontent.com/91471397/215394510-1cb6c694-30a6-4bf2-a7db-2cb99c2e07fa.png)

**Reply by imserhii** [reply]
@Bastiencc hey, can I ask you for firmware?

**Reply by Bastiencc** [reply]
[lumi.light.aqcn02_32.zip](https://github.com/Koenkk/zigbee2mqtt/files/13772707/lumi.light.aqcn02_32.zip)

**Reply by WarC0zes** [reply]
32 is better than 34 ?

**Reply by Bastiencc** [reply]
if 34 is ok for you, stay with it. I get some error with 34 , but not with 32.

**Comment by Bastiencc** [comment]
yes same things here. I use `power_outage_memory`
